src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,23 @@ const createCustomerValidator = () => {
     });
 };
 
+const listCustomerByIdValidator = () => {
+    return celebrate({
+      [Segments.PARAMS]: Joi.object().keys({
+        id: Joi.string().guid().required(),
+      }),
+    });
+};
+
+const listAllCustomersValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
Só uma sugestão, já poderia ter recebido o campo offset como um número, e também com o valor padrão `0`, caso não fosse informado.

```suggestion
            offset: Joi
                .number()
                .integer()
                .positive()
                .default(0),
```